commons/connector/upload/altoida/altoida_contrast_vision_test.avsc

```diff
@@ -0,0 +1,38 @@
+{
+    "namespace": "org.radarcns.connector.upload.altoida",
+    "name": "AltoidaContrastVisionTest",
+    "type": "record",
+    "doc": "Schema of events for the Altoida Tapping test.",
+    "fields": [
+      { "name": "time", "type": "double", "doc": "Timestamp in UTC (s) when the assessment is started by the subject." },
+      { "name": "timeReceived", "type": "double", "doc": "Timestamp in unix time received from Altoida API."},
+      { "name": "name", "type": "string", "doc": "Questionnaire names." },
+      { "name": "contrastCircles", "type": {
+        "type": "array",
+        "items": {
+          "name": "ContrastCircle",
+          "type": "record",
+          "doc": "Contrast circle screen element.",
+           "fields": [
+             { "name": "id", "type": "string", "doc": "Circle id.", "default": null },
+             { "name": "radius", "type": "double", "doc": "Circle radius" },
+             { "name": "xCenterOffset", "type": "double", "doc": "Circle location x center offset." },
+             { "name": "yCenterOffset", "type": "double", "doc": "Circle location y center offset." },
+             { "name": "color", "type": ["null", "string"], "doc": "Circle color." }
```

Comment:
what value type does this have? text? hex rgb?

Reply:
It's RGB. I've made the changes.